docroot/sites/default/settings.php

```diff
@@ -220,5 +220,4 @@
   ini_set('memory_limit', '2048M');
 }
 
-$config['entity_usage.settings']['queue_tracking'] = TRUE;
```

Comment:
I think this setting should be set per-environment, once we've run the full entity_usage_regenerate_queue queue.